tests/advanced.test.ts

```diff
@@ -172,4 +172,42 @@ describe(`Advanced`, () => {
 
         expect(output).to.equal(`Running CommandB\n"hello"\n`);
     });
+
+    it.only(`derives positional argument names from the property name`, async () => {
```

Comment:
Woops - I forgot to remove my `only` after my previous debugging session, can you remove them?
```suggestion
    it(`derives positional argument names from the property name`, async () => {
```

Reply:
I did but that caused two other tests to fail. I'll check if they work on `master`

Reply:
They were failing because of this change. All good now.